src/main/java/edu/kis/powp/command/PatternFactory.java

```diff
@@ -0,0 +1,50 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class PatternFactory {
+    public static enum Patterns {
+        RECTANGLE, TRIANGLE, LINE_VERTICAL, LINE_HORIZONTAL
+    }
+
+    private Patterns pattern;
+    private int size;
+    private Job2dDriver driver;
+
+    PatternFactory(Patterns pattern, int size, Job2dDriver driver) {
```

Comment:
very complex to use. It would be better to pass some of these parameters to getCommand directly (as command)